lib/bundler/stub_specification.rb

```diff
@@ -68,6 +68,10 @@ def raw_require_paths
       stub.raw_require_paths
     end
 
+    def to_spec
+      stub.to_spec
```

Comment:
`stub` != `self` though, one is a `Bundler::StubSpecification` and the other is `Gem::StubSpecification`.

```
18:03:57 web.1      | ===
18:03:57 web.1      | Gem::StubSpecification
18:03:57 web.1      | Bundler::StubSpecification
18:03:57 web.1      | ===